include/hri_safe_remote_control_system/VehicleMessages.h

```diff
@@ -256,6 +303,39 @@ typedef struct
   char value[VSC_USER_FEEDBACK_STRING_LENGTH];
 } UserFeedbackStringMsgType;
 
+/** SCMTargetSetMsgType
+ * 	The Structure for the packed target id that is used to transmit the SCM Target
+ *  Set command to the VSC.
+ */
+typedef struct
+{
+  uint32_t target_id;
+} SCMTargetSetMsgType;
+
+
+/** SCMTargetGetMsgType
+ * 	The Structure for the the SCM Target Get command transmitted to the VSC.
+ */
+typedef struct
+{
+} SCMTargetGetMsgType;
```

Comment:
just making sure the `SCMTargetGetMsgType` is supposed to be empty.

Reply:
It is. There's no payload to the message.